app/views/categories/new.html.erb

```diff
@@ -0,0 +1,14 @@
+<h1>Create a New Category</h1>
+<%= form_with model: @category do |form|%>  
```

Comment:
- [ ] In the Create a new category page, there should be a back button she takes the user to the Homepage (categories page). This button is currently missing
![image](https://github.com/IsmailChangezi/snap-scan/assets/62849872/9adaa5f4-08a6-4e8e-88c5-21c11ea73f26)

Reply:
* As mentioned by the previous CR kindly add a back button that takes the user back to the Homepage